src/act.drive.cpp

```diff
@@ -356,12 +356,12 @@ ACMD(do_vemote)
     return;
   }
   RIG_VEH(ch, veh)
-  snprintf(buf, sizeof(buf), "%s%s.\r\n", capitalize(GET_VEH_NAME(veh)), argument);
+  snprintf(buf, sizeof(buf), "%s%s\r\n", capitalize(GET_VEH_NAME(veh)), argument);
```

Comment:
Since we're already using a snprintf here, we may as well take advantage of an inline conditional to provide punctuation. There's a good example of this with `ispunct()` in newecho.cpp, where it's used to punctuate emotes that don't have terminal punctuation already.